src/module/addRemove.js

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable no-loop-func */
+let todoList = [];
+let isEditing = false;
+let todoEdit = null;
+
+const saveData = () => {
+  localStorage.setItem('todoslist', JSON.stringify(todoList));
+};
+
+const getStorageData = () => {
+  const localFormData = JSON.parse(localStorage.getItem('todoslist'));
+  if (localFormData == null) {
+    todoList = [];
+  } else {
+    todoList = localFormData;
+  }
+};
+
+const editTodoList = (todo) => {
+  isEditing = true;
+  todoEdit = todo;
+  const desc = document.querySelector('.taskList');
+  desc.value = todo.description;
+  desc.focus();
+};
+
+const removeTodoList = (indexID) => {
+  todoList.splice((indexID - 1), 1);
+  todoList = todoList.map(
+    (todo, index) => (
+      { completed: todo.completed, description: todo.description, index: index + 1 }
+    ),
+  );
```

Comment:
- [ ] The order of the indexes changes after deleting a task. I added 5 tasks, when I removed the fourth task, the order of indexes changed. Task 5 became with index 1 and task 1 became with index 4. I would recommend keeping the same order of tasks after deleting a task.

<img width="951" alt="image" src="https://user-images.githubusercontent.com/77190203/161181557-6000c288-0209-4d7a-a6c7-e030dc8de1b7.png">

Reply:
@medaminedev66 Will the following code will serve your purpose?

Reply:
@medaminedev66 Can you please respond?

Reply:
Hi @amrendrakind, sorry for the late reply. Yes exactly, deleting a task should update all remaining items' indexes, so they represent **the current list order** and are unique.